keps/sig-auth/20190607-certificates-api.md

```diff
@@ -379,9 +380,31 @@ that uses a webhook to sign all approved CSRs. This allows the root certificate
 authority secret material to be stored and maintained outside of the Kubernetes
 control plane.
 
+### Test Plan
+
+- unit tests of:
+  - API defaulting
+  - API validation
+  - API round-tripping
+  - Authorizing admission plugin
+  - Auto-approving controller
+  - Auto-signing controller
+
+- integration tests of:
+  - auto-approve and signing of kubelet client CSR
+  - manual-approve and auto-signing of kubelet serving CSR
+  - manual-approve and auto-signing of API client CSR
+  - manual-approve and auto-signing of legacy CSR
+
+- e2e (and conformance, once v1) tests of:
+  - CSR API presence, basic crud behavior
+  - status subresource
+  - approval flow
```

Comment:
Approval as general concept isn't owned by kubernetes.  Neither is signing.  An e2e test for what we ship makes a lot of sense.  I'm less convinced that non-kube owned code should have conformance tests.

Reply:
scoped to a custom signerName, to ensure the mechanics work as designed and are usable by other approvers/signers